bucket/appium-inspector.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "2023.12.2",
+    "description": "A GUI inspector for mobile apps and more, powered by a (separately installed) Appium server",
+    "homepage": "https://github.com/appium/appium-inspector",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/appium/appium-inspector/releases/download/v2023.12.2/Appium-Inspector-windows-2023.12.2-x64.zip",
+            "hash": "4d361d52866f2eb2ca5c4e796180d8d1d99d0a354fee43ced30046fc47ba3063"
+        },
+        "32bit": {
+            "url": "https://github.com/appium/appium-inspector/releases/download/v2023.12.2/Appium-Inspector-windows-2023.12.2-ia32.zip",
+            "hash": "72ea596f461001d863e54c25a15aa01bc68b884d5ad25eaec9cbd76fa7547670"
+        }
+    },
+    "bin": "Appium Inspector.exe",
```

Comment:
```suggestion
```

Suppose this could be removed since it's GUI app and there is no CLI mode/option.
Otherwise looks good.